src/CollectdWinService/config/WriteHttpPluginConfig.cs

```diff
@@ -54,6 +54,27 @@ public int MaxIdleTime
                 set { base["MaxIdleTime"] = value; }
             }
 
+            [ConfigurationProperty("UserName", IsRequired = false)]
+            public string UserName
+            {
+                get { return (string)base["UserName"]; }
+                set { base["UserName"] = value; }
+            }
+
+            [ConfigurationProperty("Password", IsRequired = false)]
+            public string Password
+            {
+                get { return (string)base["Password"]; }
+                set { base["Password"] = value; }
+            }
+
+            [ConfigurationProperty("UseSafeNames", IsRequired = false)]

```

Comment:
Thanks, and I'll also do the right thing and use RegEx.  I'll create the RegEx once on config and it won't be expensive. It looks like there have been a bunch of other updates to master, but not to these files. Thx again for the help.